deploy/demo/dataset-catalog-frontend/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
demo er litt for forskjellig fra staging og prod
- mangler env
- mangler ingress
- bruker ikke base